android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt

```diff
@@ -118,16 +142,30 @@ class ScreenStackFragment : ScreenFragment, ScreenStackFragmentWrapper {
             // role. On top of that it breaks screens animations when alfa offscreen compositing is off
             // (which is the default)
             setBackgroundColor(Color.TRANSPARENT)
+
             layoutParams = AppBarLayout.LayoutParams(
-                AppBarLayout.LayoutParams.MATCH_PARENT, AppBarLayout.LayoutParams.WRAP_CONTENT
+                AppBarLayout.LayoutParams.MATCH_PARENT, getHeightOfToolbar(context)
             )
+
+            // On Material 3 the elevation is not visible on AppBarLayout.
+            // To prevent this behavior, we're setting outline shadow colors to black.
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+                outlineAmbientShadowColor = Color.BLACK
+                outlineSpotShadowColor = Color.BLACK
+            }
+
+            fitsSystemWindows = true
```

Comment:
What does this flag do?

Reply:
It was used to render the content of the view, excluding the padding of system native elements (like status bar, navigation bar). Hmm, I believe I had some issues with app bar layout without this property, but I can't see any problems at the moment, when I'm removing this prop. I'll remove it 👍

Reply:
Resolved in https://github.com/software-mansion/react-native-screens/pull/1874/commits/4c5612a3a6acc7788da2585a399970ace5dd6124.